resources/template.yaml

```diff
@@ -23,7 +23,6 @@ parameters:
 # Parameters with no defaults (has to be be provided when template is applied):
 - name: ROUTE_HOSTNAME
   description: The external hostname to access Syndesis
-  required: true
```

Comment:
the problem with removing this (when we move it over to syndesis for now) is, that the template is not really usable outside of the operator anymore. 

Longterm that template *should* not be used outside anymore anyway (it becomes an implementation detail of the operator), but for now it maybe a bit early to remove it.

What's about using a initial dummy/marker value for the host when processing the template (so that 'require' is happy), then create the route and replace that value for the second processing of the templates ?